src/main/paradox/docs/api/iam/authentication.md

```diff
@@ -0,0 +1,7 @@
+# Authentication
+
+In order to interact with Nexus, clients need a valid access token. Tokens can be obtained from authentication providers.
+Please see, @ref:[realms](iam-realms-api.md) for documentation on how to find available providers(realms). Each realm defines
```

Comment:
This block seems quite compacted. I would add 2 newline before `Each realm defines...` so that it appears in a new row.